decidim-core/lib/decidim/form_builder.rb

```diff
@@ -157,6 +157,33 @@ def scopes_picker(attribute, options = {})
       template.html_safe
     end
 
+    # Public: Generates a picker field for selection (either simple or multiselect).
+    #
+    # attribute     - The name of the object's attribute.
+    # options       - A Hash with options:
+    # - multiple: Multiple mode, to allow selection of multiple items.
+    # - label: Show label?
+    # - name: (optional) The name attribute of the input elements.
+    # prompt_params - Hash with options:
+    # - url: The url where the ajax endpoint that will fill the content of the selector popup (the prompt).
+    # - text: Text in the button to open the Data Picker selector.
+    #
+    # Also it should receive a block that returns a Hash with :url and :text for each selected scope
+    #
+    # Returns an html String.
+    def data_picker(attribute, options = {}, prompt_params = {})
+      picker_options = { id: "#{@object_name}_#{attribute}", class: "picker-#{options[:multiple] ? "multiple" : "single"}",
```

Comment:
Can you normalize this hash please? Each key in a single line, etc

Reply:
Note that rubocop explicitly [accepts this style](https://github.com/wata727/rubocop/blob/f40a9763541cc87e53bf8fa97b8947677211a583/spec/rubocop/cop/layout/align_hash_spec.rb#L18-L23) and this file is using that same style in other places, so I think it's fine to leave it since otherwise we're introducing "in-file" inconsistency.

As a note, I made a very related [feature request](https://github.com/bbatsov/rubocop/issues/5184) to rubocop a while ago, but nobody has stepped to implement it. It's not even clear if it should be a different cop or a configuration option for existing cops.